tests/test_common.h

```diff
@@ -30,7 +30,7 @@
   #include <sys/time.h>
 #endif
 
-#if defined(_WIN32) && !defined(__MINGW32__)
+#if defined(_WIN32)
```

Comment:
Just curious: setenv is not in MINGW32 either?

Reply:
From my understanding mingw32 uses Windows API. There is a discussion recently about this on [zlib-ng](https://github.com/zlib-ng/zlib-ng/pull/566). I made the change because it was failing on `Windows GCC` CI run which is viewable [here](https://github.com/nmoinvaz/c-blosc2/runs/620807821).

```
tests/test_nthreads.c: In function 'main':
tests/test_nthreads.c:87:3: warning: implicit declaration of function 'setenv'; did you mean 'getenv'? [-Wimplicit-function-declaration]
    setenv("BLOSC_NTHREADS", "3", 1);
    ^~~~~~
    getenv
...
tests/CMakeFiles/test_nthreads.dir/test_nthreads.c.obj:test_nthreads.c:(.text.startup+0x21): undefined reference to `setenv'
collect2.exe: error: ld returned 1 exit status
```